templates/base.html

```diff
@@ -290,6 +290,26 @@ <h3>{{config.page_title.prefix}} Sponsors</h3>
       $('.dropdown-menu a.dropdown-toggle').click(ddHoverOrClick).hover(ddHoverOrClick).on('touchstart', ddHoverOrClick);
     });
   </script>
+
+  <!-- Prevent focus from jumping to embedded rocket.chat with an iframe id="chatFrame" -->
+  if ($("#chatFrame").val() != "") {
+    <script type="text/javascript">
+        $("#chatFrame").hide();
+        $("#chatFrame").on(
+          "load",
+          function() {
+            setTimeout(
```

Comment:
Am I correctly inferring that this is basically just adding a 3 second delay on loading the RocketChat pane? If yes, could we reduce it? I tested locally and with 1500 instead of 3000 it doesn't seem to steal focus, although I'm wondering if there's a browser-specific thing happening here.

Reply:
@corcra yes, it is preventing the iframe from showing until it is loaded. There is likely some variation in the loading speed, so I have stuck with the 3000 ms as discussed in more detail here: https://github.com/acl-org/acl-2020-virtual-conference/issues/138#issuecomment-647398289 .